PBM-570 A config file options page (#442)
percona/percona-backup-mongodb: doc/source/index.rst

.. note::

>    |PBM| doesn't work on standalone |mongodb| instances. This is because |PBM| requires an :term:`oplog <Oplog>` to guarantee backup consistency. Oplog is available in clusters and replica sets only.

Reviewer: ```suggestion
   |PBM| doesn't work on standalone |mongodb| instances. This is because |PBM| requires an :term:`oplog <Oplog>` to guarantee backup consistency. Replication must be enabled.
```
I think it must be confusing to talk about clusters, it is an concept orthogonal to replication.
